beacon-chain/execution/service_test.go

```diff
@@ -82,7 +84,7 @@ func (g *goodNotifier) StateFeed() *event.Feed {
 
 type goodFetcher struct {
 	backend     *backends.SimulatedBackend
-	blockNumMap map[uint64]*gethTypes.Header
+	blockNumMap map[uint64]*pb.ExecutionBlock
```

Comment:
This one is different than the mock rpcClient?

```go
// beacon-chain/execution/testing/mock_execution_chain.go
// RPCClient defines the mock rpc client.
type RPCClient struct {
	Backend *backends.SimulatedBackend
	Backend     *backends.SimulatedBackend
	BlockNumMap map[uint64]*types.HeaderInfo // not pb.ExecutionBlock
}
```

Also, can you just use that mock RPCClient?

Reply:
We don't need the whole mock actually, just deleting it now